mergify_engine/github_types.py

```diff
@@ -475,3 +475,15 @@ class GitHubEventTeam(GitHubEvent):
 class GitHubEventTeamAdd(GitHubEvent, total=False):
     # Repository key can be missing on Enterprise installations
     repository: GitHubRepository
+
+
+GitHubGitRefType = typing.NewType("GitHubGitRefType", str)
```

Comment:
This already exists AFAIK

Reply:
`GitHubGitRefType` != `GitHubRefType`, its Git Database API vs Branches API.

And they should not be mixed as the first one has git namespace inside (eg: refs/heads/...)